python/docs/additions/compute/all.rst

```diff
@@ -0,0 +1,24 @@
+all
+===
+
+Examples
+--------
+
+.. code-block:: python
```

Comment:
For our other docstrings (in the code), we do this without the `code-block:: python` (which doctest supports, and sphinx as well to render). I suppose that can be done here as well?

See eg https://github.com/apache/arrow/blob/b832853ba62171d5fe5077681083fc6ea49bfd44/python/pyarrow/array.pxi#L180-L185

Reply:
So it does! I didn't realize that.